docs/architecture/adr-006-upgrade-client-implementation.md

```diff
@@ -131,7 +131,7 @@ is entirely handled.
 
 1. Upgrade chain through governance
    1. Create a 02-client
-      [UpgradeProposal](https://github.com/cosmos/ibc-go/blob/main/docs/ibc/proto-docs.md#upgradeproposal)
+      [UpgradeProposal](https://github.com/cosmos/ibc-go/blob/main/docs/docs/01-ibc/09-proto-docs.md#upgradeproposal)
```

Comment:
The v7 link goes to a doc page with a single link, and that single link is broken 😅

So it's a bit weird, because the v6.2.1 link does go to a functioning doc page, so not sure whether updating this link is worthwhile.

Reply:
Sounds good. I linked to the `main` in my PR. It should be fine if the correct link is maintained on `ibc-go`'s latest branch.